lib/semian/simple_enum.rb

```diff
@@ -0,0 +1,29 @@
+module Semian
+  module Simple
+    class Enum #:nodoc:
+      def initialize(symbol_list:)
```

Comment:
yeah, but it's so key to this class and no default makes sense. It doesn't matter much. Up to you.